.devcontainer/docker-compose.yml

```
@@ -0,0 +1,34 @@
+version: '3.9'
+
+networks:
+  local_network:
```

To prevent conflicts with other projects?

```suggestion
  local_credo_mediator_network:
```